scripts/schema/cleaner.py

```diff
@@ -234,3 +236,21 @@ def single_line_beta_description(schema_or_field, strict=True):
             raise ValueError(msg)
         else:
             ecs_helpers.strict_warning(msg)
+
+
+def single_line_short_override_description(schema_or_field, strict=True):
+    for field in schema_or_field['schema_details']['reusable']['expected']:
+        if not 'short_override' in field:
+            continue
+        short_length = len(field['short_override'])
+        if "\n" in field['short_override'] or short_length > SHORT_LIMIT:
```

Comment:
An observation, but not a blocking issue to merging this as is.

Might be an opportunity to refactor and combine what's happening here and `single_line_short_description` since there's some duplication.